### PR TITLE
Fixed table

### PR DESCRIPTION
**Pull request by NicoInve**
Fixed CRS305-1G-4S+IN Type.
Added CRS305-1G-4S+IN, CCR2004-1G-12S+2XS and CCR2116-12G-4S+ vendor.

### CODE REVIEW

**Comment by simonebortolin** [comment]
Please fix the white space in the table

File: _sfp_cage/mikrotik.md [hunk 1]
```diff
@@ -8,14 +8,14 @@ layout: default
...
-| Vendor   | MikroTik        |                 |                    |                 |
+| Vendor   | MikroTik        | Mikrotik        | Mikrotik           | Mikrotik        |
 | Model    | RB5009          | CRS305-1G-4S+IN | CCR2004-1G-12S+2XS | CCR2116-12G-4S+ |
 | SFP      | 1 SFP+          | 4 SFP+          | 12 SFP+, 2 SFP28   | 4 SFP+          |
 | Ethernet | 1 2.5GbE, 7 GbE | 1 GbE           | 1 GbE              | 13 GbE          |
 | XGMII    | ✅              | ✅              | ✅                 | ✅              |
 | HSGMII   | ✅              | ✅              | ✅                 | ✅              |
 | SGMII    | ✅              | ✅              | ✅                 | ✅              |
-| Type     | Router          | Switch          | Switch             | Router          |
+| Type     | Router          | Switch         | Router            | Router          |
```
Comment (simonebortolin) [inline _sfp_cage/mikrotik.md:18]:
```suggestion
| Type     | Router          | Switch         | Router             | Router           |
```

**Review by simonebortolin — CHANGES_REQUESTED** [review]

**Review by nanomad — APPROVED** [review]

File: _sfp_cage/mikrotik.md (hunk 1 above)
Comment (simonebortolin) [reply _sfp_cage/mikrotik.md:18]: Please fix or apply suggestion.

File: _sfp_cage/mikrotik.md [hunk 2]
```diff
@@ -8,14 +8,14 @@ layout: default
...
-| Vendor   | MikroTik        |                 |                    |                 |
+| Vendor   | MikroTik        | Mikrotik        | Mikrotik           | Mikrotik        |
 | Model    | RB5009          | CRS305-1G-4S+IN | CCR2004-1G-12S+2XS | CCR2116-12G-4S+ |
 | SFP      | 1 SFP+          | 4 SFP+          | 12 SFP+, 2 SFP28   | 4 SFP+          |
 | Ethernet | 1 2.5GbE, 7 GbE | 1 GbE           | 1 GbE              | 13 GbE          |
 | XGMII    | ✅              | ✅              | ✅                 | ✅              |
 | HSGMII   | ✅              | ✅              | ✅                 | ✅              |
 | SGMII    | ✅              | ✅              | ✅                 | ✅              |
-| Type     | Router          | Switch          | Switch             | Router          |
+| Type     | Router          | Switch         | Router             | Router           |
```
Comment (simonebortolin) [inline _sfp_cage/mikrotik.md:18]:
```suggestion
| Type     | Router          | Switch          | Router             | Router          |
```

**Review by simonebortolin — CHANGES_REQUESTED** [review]

**Pull request merged**